services/requester/batch_tx_pool.go

```diff
@@ -123,6 +126,19 @@ func (t *BatchTxPool) Add(
 		return err
 	}
 
+	eoaActivity, found := t.eoaActivityCache.Get(from)
+	nonce := tx.Nonce()
+
+	// Reject transactions that have already been submitted,
+	// as they are *likely* to fail.
+	if found && slices.Contains(eoaActivity.txNonces, nonce) {
+		return fmt.Errorf(
```

Comment:
should we return an error here, or just silently drop the tx? What happens on a geth node?